newIDE/app/src/ProjectManager/ProjectManagerMainMenu.js

```diff
@@ -0,0 +1,241 @@
+        {isNavigatingInsideSubMenu && (
+          <Line noMargin expand alignItems="flex-start">
+            <TextButton
+              icon={<ChevronArrowLeft />}
+              label={<Trans>Back</Trans>}
+              onClick={onBack}
+            />
+          </Line>
+        )}
+        {/* Root items as buttons */}
+        {!isNavigatingInsideSubMenu && (
+          <Line noMargin expand>
+            {mainMenuItems.map((item, index) => {
+              if (item.type === 'separator') return null;
+              // Use a "native" or "simulated native" menu on desktop & web, or handle navigation on mobile
+              return (
+                <React.Fragment key={index}>
+                  {index !== 0 && (
+                    <>
+                      <Spacer />
+                      <div className={classes.verticalSeparator} />
+                      <Spacer />
+                    </>
+                  )}
+                  {shouldUseNativeMenu ? (
+                    <ElementWithMenu
+                      element={
+                        <TextButton
+                          label={item.label}
+                          onClick={() => {
+                            // Handled by ElementWithMenu
+                          }}
+                          fullWidth
+                        />
+                      }
+                      buildMenuTemplate={() => onBuildSubMenuTemplate(index)}
+                    />
+                  ) : (
+                    <TextButton
+                      label={item.label}
+                      onClick={() => {
+                        // Assume root items always have a submenu.
+                        setSelectedMainMenuItemIndices([
+                          ...selectedMainMenuItemIndices,
+                          index,
+                        ]);
+                      }}
+                      fullWidth
+                    />
+                  )}
+                </React.Fragment>
+              );
+            })}
+          </Line>
+        )}
+        {/* Submenu items as rows, which will be visible only on mobile */}
+        {isNavigatingInsideSubMenu && (
+          <Column expand>
+            {visibleMenuItems.map((item, index) =>
+              item.type === 'separator' ? (
+                <div key={index} className={classes.horizontalSeparator} />
+              ) : (
+                <div
+                  className={classNames({
+                    [classes.menuItem]: true,
+                    [classes.disabled]: !item.submenu && item.enabled === false,
+                  })}
+                  key={index}
+                  onClick={() => {
+                    if (item.enabled === false) return;
+
+                    if (item.click) {
+                      item.click();
+                      // Close the drawer after clicking on a menu item,
+                      // so the user can see the result of the action.
+                      closeDrawer();
```

Comment:
Isn't this closeDrawer already called thanks to enhanceMenuItems ?

Reply:
you may be right

Reply:
actually it wasn't but I'm fixing it by calling that function properly for both menus